README.md

```diff
@@ -12,12 +12,15 @@ LibreOffice SDK is not required.
 
 To build:
 
-1.  Run `scripts/symlink_sdk`
+1.  Run `scripts/symlink_sdk`.
 1.  Open Eclipse and import this project into your workspace.
-1.  Right-click the project in the Eclipse Package Explorer, click "Properties", and then select "Java Build Path." Click the libraries tab and ensure that all referenced files exist, or else correct the paths.
+1.  Right-click the project in the Eclipse Package Explorer, click "Properties", select "Java Build Path", then select the "Libraries" tab. Click on "Add JARs" and expand the "Zotero OpenOffice Integration" folder. Make sure there is no "lib" subfolder underneath it. If there is:
+	1. Expand "Zotero OpenOffice Integration" -> "lib" -> "libreoffice-sdk" and select all the JARs underneath it.
+	1. Click "Ok" and then "Apply and Close".
```

Comment:
This is not something that needs to be done. The plugin compiles fine without this step. Could you expand why you want to add this?